packages/css/src/components/breakout/breakout.scss

```diff
@@ -3,105 +3,105 @@
  * Copyright Gemeente Amsterdam
  */
 
-@import "../../common/breakpoint";
-@import "../grid/grid";
-@import "../grid/mixins";
+@use "../../common/breakpoint";
+@use "../grid/grid";
+@use "../grid/mixins";
 
 $ams-breakout-row-span-max: 4;
 
 .ams-breakout {
-  @include ams-grid;
+  @include mixins.ams-grid;
 }
 
 // Grid gap
 
 .ams-breakout--gap-vertical--none {
-  @include ams-grid--gap-vertical--none;
+  @include mixins.ams-grid--gap-vertical--none;
 }
 
 .ams-breakout--gap-vertical--small {
-  @include ams-grid--gap-vertical--small;
+  @include mixins.ams-grid--gap-vertical--small;
 }
 
 .ams-breakout--gap-vertical--large {
-  @include ams-grid--gap-vertical--large;
+  @include mixins.ams-grid--gap-vertical--large;
 }
 
 // Grid padding
 
 .ams-breakout--padding-bottom--small {
-  @include ams-grid--padding-bottom--small;
+  @include mixins.ams-grid--padding-bottom--small;
 }
 
 .ams-breakout--padding-bottom--medium {
-  @include ams-grid--padding-bottom--medium;
+  @include mixins.ams-grid--padding-bottom--medium;
 }
 
 .ams-breakout--padding-bottom--large {
-  @include ams-grid--padding-bottom--large;
+  @include mixins.ams-grid--padding-bottom--large;
 }
 
 .ams-breakout--padding-top--small {
-  @include ams-grid--padding-top--small;
+  @include mixins.ams-grid--padding-top--small;
 }
 
 .ams-breakout--padding-top--medium {
-  @include ams-grid--padding-top--medium;
+  @include mixins.ams-grid--padding-top--medium;
 }
 
 .ams-breakout--padding-top--large {
-  @include ams-grid--padding-top--large;
+  @include mixins.ams-grid--padding-top--large;
 }
 
 .ams-breakout--padding-vertical--small {
-  @include ams-grid--padding-vertical--small;
+  @include mixins.ams-grid--padding-vertical--small;
 }
 
 .ams-breakout--padding-vertical--medium {
-  @include ams-grid--padding-vertical--medium;
+  @include mixins.ams-grid--padding-vertical--medium;
 }
 
 .ams-breakout--padding-vertical--large {
-  @include ams-grid--padding-vertical--large;
+  @include mixins.ams-grid--padding-vertical--large;
 }
 
 // Column span
 // The order of these declaration blocks ensures the intended specificity.
 
 .ams-breakout__cell--col-span-all {
-  @include ams-grid__cell--span-all;
+  @include mixins.ams-grid__cell--span-all;
 }
 
-@for $i from 1 through $ams-grid-column-count {
+@for $i from 1 through grid.$ams-grid-column-count {
   .ams-breakout__cell--col-span-#{$i} {
-    @include ams-grid__cell--span($i);
+    @include mixins.ams-grid__cell--span($i);
   }
 
   .ams-breakout__cell--col-start-#{$i} {
-    @include ams-grid__cell--start($i);
+    @include mixins.ams-grid__cell--start($i);
   }
 }
 
-@media screen and (min-width: $ams-breakpoint-medium) {
-  @for $i from 1 through $ams-grid-column-count {
+@media screen and (min-width: breakpoint.$ams-breakpoint-medium) {
```

Comment:
We could try to lose one of the two mentions of `breakpoint`.

Reply:
Do they need the prefix? `breakpoints.$medium` looks better.

Reply:
It depends on whether we consider the mixins public API and whether they may introduce conflicts if people use our library together with others. If either is true, we should add a prefix for functional and technical reasons. If not, we can consider esthetic motives.